nilearn/masking.py

```diff
@@ -513,6 +513,139 @@ def compute_multi_background_mask(data_imgs, border_size=2, upper_cutoff=0.85,
     return mask
 
 
+def compute_gray_matter_mask(target_img, threshold=1,
+                             connected=True, opening=2,
+                             interpolation='nearest', verbose=0):
+    """ Compute a mask corresponding to the gray matter part of the brain.
+    The gray matter part is found through the resampling of MNI152 template
+    from which the range of values for GM is known
+
+    Parameters
+    ----------
+    target_img: Niimg-like object
+        See http://nilearn.github.io/manipulating_images/input_output.html
+        Images used to compute the mask. 3D and 4D images are accepted.
+        If a 3D image is given, we suggest to use the mean image
+
+    threshold: float, optional
+        The value under which the MNI template is cut off.
+
+    connected: boolean, optional
+        if connected is True, only the largest connect component is kept.
+
+    opening: bool or int, optional
+        if opening is True, a morphological opening is performed, to keep
+        only large structures. This step is useful to remove parts of
+        the skull that might have been included.
+        If opening is an integer `n`, it is performed via `n` erosions.
+        After estimation of the largest connected constituent, 2`n` closing
+        operations are performed followed by `n` erosions. This corresponds
+        to 1 opening operation of order `n` followed by a closing operator
+        of order `n`.
+
+    interpolation: str, optional
+        Specify the resampling strategy. The mask being binary (1=GM, 0=non GM)
+        it can be better to chose interpolation='nearest' or 'linear'
+        The threshold has to be lower than 1 if resampling='continuous'
```

Comment:
I wouldn't expose this option: I think that the interpolation should really be "nearest".